src/main/java/uk/gov/homeoffice/digital/sas/person/model/Person.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.homeoffice.digital.sas.person.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.validation.constraints.DecimalMax;
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import uk.gov.homeoffice.digital.sas.jparest.annotation.Resource;
+import uk.gov.homeoffice.digital.sas.jparest.models.BaseEntity;
+import uk.gov.homeoffice.digital.sas.person.enums.TermsAndConditions;
+
+@Resource(path = "persons")
+@Entity(name = "person")
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@NoArgsConstructor
+@Getter
+@Setter
+public class Person extends BaseEntity {
+
+  @NotNull(message = "Person Version should not be empty")
+  @Column(name = "version")
+  private Integer version;
+
+  @NotNull(message = "Person First Name should not be empty")
+  @Column(name = "first_name")
+  private String firstName;
```

Comment:
nit: I don't think we actually need to specify the column names as the spring naming stratergy should automatically convert the name to the database convention. E.g. firstName would become first_name.

Not sure why it still exists in timecard, but since this is new we could possibly do without it. Unless we have a good reason for it / general preference